producer/src/main/kotlin/streams/RoutingConfiguration.kt

```diff
@@ -226,6 +227,20 @@ data class RelationshipRoutingConfiguration(val name: String = "",
     }
 }
 
+data class RelKeyStrategyConfiguration(val topic: String = "neo4j", val strategy: RelKeyStrategy= RelKeyStrategy.first) {
```

Comment:
io credo sia meglio mettere semplicemente `val strategy: RelKeyStrategy = RelKeyStrategy.first` dentro `RelationshipRoutingConfiguration` visto che stiamo duplicando molte delle informazioni, che dici?

Reply:
Ok, ci avevo pensato anch'io però visto che `key_strategy` è di fatto un'altra configurazione rispetto al `streams.source.topic.relationships.<TOPIC_NAME>` ho pensato di metterlo in un'altra `data class`. Però sì, se va bene anche così meglio, si semplifica un bel po'.